span-key-extractor/src/main/java/com/expedia/www/haystack/pipes/commons/key/extractor/Constants.java

```diff
@@ -0,0 +1,9 @@
+package com.expedia.www.haystack.pipes.commons.key.extractor;
+
+public interface Constants {
```

Comment:
This should not be an interface. Interfaces are for defining types. Should be either enums returning strings or a final class perhaps.